### ISSUE TITLE
Populate extra data into planter and organization and wallet

### ISSUE THREAD

**Issue by dadiorchen** [Greenstand/treetracker-query-api#144]
To add more extra info to the API:
GET /planters
GET /planters/{id}
GET /organizations
GET /organizations/{id}
GET /wallets
GET /wallets/{id}

The background: now we are building the stakeholder management panel on the admin panel team, basically, the org and planter's `about` and `featured trees` will come from there, but we  will not wait for their release, instead, we will use a temporary solution to display these info on our web map.

The solution: in database `webmap` schema, the table `config` is for this goal, the table:

```
CREATE TABLE webmap.config (
    name character varying(255) NOT NULL,
    data json NOT NULL,
    ref_id integer,
    ref_uuid uuid
);

```

we will dump data to this table, and put the entity id in `ref_id` column.

The data for organization:
```
name: organization-extra
ref_id: [entity_id of this org, for example: 178]
data: {
  about: [about information],
  mission: [mission information],
  featured_tree: [id1, id2...]
```

Basically, we just need to fetch the data by the id and fixed name (organization-extra here), then add all these fields into the object that are going to return to client.
like:
{
  id: xxx,
  name: org name,
  about: ...
  mission: ...
}

And the planter and wallet are the same.

Please also consider the list case, say, `GET /planters?filter=xxx`

Feel free to add data into the db table for testing.

**Comment by dadiorchen** [comment]
To be aware that this issue is kind of duplicated with https://github.com/Greenstand/treetracker-query-api/issues/139

**Comment by dadiorchen** [comment]
@waynelao this one is a bit urgent now, I will help on this and raise a PR soon
